### ISSUE TITLE
[luis.ai] Submit utterance under "Suggest" menu error: "Failed to add utterance."

### ISSUE THREAD

**Issue by zixia** [Microsoft/BotBuilder#476]
I can see all the utterance at the web interface(luis.ai) after I use luis api to query(by LuisDialog.js).

But some query is quite long(about 320 chinese utf8 charactors), and when I submit it under the suggest menu, the web interface said:
> Failed to add utterance.

I check the doc and found this: 

> Note that sentences longer than 500 characters will result in an error message.

at here: https://www.luis.ai/Help#PublishingModel

but I think my utterance is not long enough to exceed the 500 characters limit. 

then what's the wrong with the web interface? is there because the 500 is byte, not utf8-char? 

thanks.

**Comment by willportnoy** [comment]
Sorry, I don't know the answer to your question.  You might find an expert on the LUIS forum at https://social.msdn.microsoft.com/Forums/azure/en-US/home?forum=LUIS.

**Comment by zixia** [comment]
I got the error again. and It seems the byte size limitation is 500.

I use this code snip to get the substr no more then 500 byte size. (stupid code but works, because I'm not familiar with calculating the utf8 encoded chars...)

```javascript
    let byteSize = Buffer(utterance).length

    while (byteSize > 500) {
      utterance = utterance.substr(0, Math.floor(utterance.length * 500 / byteSize))
      byteSize = Buffer(utterance).length
    }
```

And another issue is: if I got a error utterance in the luis.ai web UI, then I will never get rid of that. I must export the module, delete the error utterance in the json file, delete the module, and re-import the module again to fix that.
